clang/include/clang/Basic/DiagnosticSemaKinds.td

```diff
@@ -10121,10 +10121,10 @@ def err_sycl_attibute_cannot_be_applied_here
     : Error<"%0 attribute cannot be applied to a "
             "%select{static function or function in an anonymous namespace"
             "|class member function}1">;
-def warn_sycl_attibute_address_space_ext
+def warn_sycl_attibute_function_raw_ptr
     : Warning<"SYCL 1.2.1 specification does not allow %0 attribute applied "
-              "to a %select{function with a raw pointer return type"
-              "|function with a raw pointer parameter type}1">,
+              "to a function with a raw "
+              "%select{pointer return type|pointer parameter type}1">,
```

Comment:
Is there any reason to duplicate "pointer" in the both options?
If not, please, move it outside of the selector.

Reply:
Done.